lib/Sema/CSApply.cpp

```diff
@@ -2190,23 +2186,17 @@ namespace {
       //   """
```

Comment:
You should probably update `interpolationInitCall` in this comment to `interpolationExpr` or whatever its final name is.